scripts/check_ecosystem.py

```diff
@@ -294,9 +295,47 @@ async def main(*, ruff1: Path, ruff2: Path, projects_jsonl: Optional[Path]) -> N
                 print()
                 print("</p>")
                 print("</details>")
+
+                # Count rule changes
+                for line in diff_str.splitlines():
+                    # Find rule change for current line or construction
+                    # + <rule>/<path>:<line>:<column>: <rule_code> <message>
+                    matches = re.findall(r": [A-Z]{1,3}[0-9]{3,4}", line)
```

Comment:
just finding the first match is sufficient here

Reply:
Addressed in 9a3c499fbbbcc3b422362dddcdfe40146e665146